system/lib/wasmfs/syscalls.cpp

```diff
@@ -861,6 +861,39 @@ int __syscall_rmdir(intptr_t path) {
   return __syscall_unlinkat(AT_FDCWD, path, AT_REMOVEDIR);
 }
 
+// wasmfs_unmount is similar to __syscall_unlinkat, but assumes AT_REMOVEDIR is true
+// and will only unlink mountpoints (Empty and nonempty).
+int wasmfs_unmount(int dirfd, intptr_t path) {
```

Comment:
We should either call this `wasmfs_unmountat` or remove the `dirfd` arg, which only appears in "at" versions of syscalls.

Reply:
I think the `dirfd` arg might not be necessary, but I like `wasmfs_unmountat` to kind of differentiate the syscall from `_wasmfs_unmount` in `js_api.cpp`. Is it reasonable to continue using `dirfd` just for the sake of the name?